python/lsst/ap/association/assoc_db_sqlite.py

```diff
@@ -181,12 +180,11 @@ def source_record_to_value_list(self, source_record, obj_id=None):
             if sub_schema.getField().getTypeString() == 'Angle':
                 values.append(
                     source_record.get(sub_schema.getKey()).asDegrees())
-            elif obj_id is not None and \
-                 sub_schema.getField().getName() == "diaObjectId":
+            elif obj_id is not None and sub_schema.getField().getName() == "diaObjectId":
                 # diaObjectId is a special case that will not be found in a
                 # difference catalog returned from the Butler. Hence we
                 # explicitly set it here.
-                values.append(obj_id)
+                values.append(int(obj_id))
```

Comment:
Dare I ask what other type `obj_id` could be?

(Also, my usual question about what if the original record *does* have a diaObjectId.)

Reply:
This was another instance of the numpy return type not playing well with some of the wrapped python wrapped afw code.